client-web/test/exlibris-app/exlibris-app_store_test.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>exlibris-app store test</title>
+
+    <script src="../../bower_components/webcomponentsjs/webcomponents-loader.js"></script>
+    <script src="../../bower_components/web-component-tester/browser.js"></script>
+    <script src="../../node_modules/redux/dist/redux.js"></script>
+    <script src="test_variables.js"></script>
+
+
+    <link rel="import" href="../../src/exlibris-app/exlibris-app.html">
+</head>
+<body>
+
+<test-fixture id="BasicTestFixture">
+    <template>
+        <exlibris-app></exlibris-app>
+    </template>
+</test-fixture>
+
+<script>
+    suite('exlibris-app', function () {
+
+        test('should set allBooks value from state', function () {
+            const mock = sinon.mock(ExlibrisApplication.store);
```

Comment:
`const mock = sinon.mock(ExlibrisApplication.store);`
and 
`mock.restore();`
are used in each test. We can use `setup` and `teardown` instead.